src/main/java/org/apache/mesos/hdfs/state/AcquisitionPhase.java

```diff
@@ -6,5 +6,5 @@
   START_NAME_NODES, // Launches the both namenodes
   FORMAT_NAME_NODES, // Formats both namenodes (first with initialize, second with bootstrap
   DATA_NODES // If everything is healthy the scheduler stays here and tries to launch
-             // datanodes on any slave that doesn't have an hdfs task running on it
+  // datanodes on any slave that doesn't have an hdfs task running on it
```

Comment:
This formatting change looks a little awkward to me, but I don't care too much. It's certainly a special case, and the benefits of global formatting rules outweigh the need for special multi-line end-of-line comments.